proto/proto-backplane-grpc/build.gradle

```diff
@@ -18,7 +18,7 @@ configurations {
 }
 
 // TODO(deephaven-core#1685): Create strategy around updating and maintaining protoc version
```

Comment:
See the notes here too, https://github.com/deephaven/deephaven-core/issues/1685 - we probably need to update the base images.

Reply:
Here: https://github.com/deephaven/deephaven-base-images/pull/20